certificate-authority/config.yaml

```diff
@@ -27,7 +27,7 @@ apis:
       certFile: "/secrets/private/cert.crt"
       clientCertificateRequired: true
     authorization:
-      ownerClaim: "sub"
```

Comment:
Why did you remove this from example `config.yaml`? Please revert.

Reply:
I would rather remove this here: https://github.com/plgd-dev/hub/blob/main/charts/plgd-hub/values.yaml#L436